1-exercises/B-while-loop/exercise.js

```diff
@@ -1,12 +1,18 @@
 /*
     while loops can be useful when you want to execute some code as long as some condition is true.    
 
-    Using a while loop, complete the function below so it logs (using console.log) the first n even numbers as a comma-seperated string.
+    Using a while loop, complete the function below so it logs (using console.log) 
+    the first n even numbers as a comma-seperated string.
     The list of numbers should start with 0. n is being passed in as a parameter.
 */
 
 function evenNumbers(n) {
-    // TODO
+  // TODO
+  let i = 0;
+  while (i % 2 === 0) {
```

Comment:
So, this looks good from a distance but it doesn't quite work as intended. The loop will only run once whatever number you pass to the function. Can you see why?